Extensions/Manage/Dnn.PersonaBar.Sites/Components/SitesController.cs

```diff
@@ -425,10 +425,11 @@ public string ExportPortalTemplate(ExportTemplateRequest request, UserInfo userI
             //Validate Home Folder
             if (!string.IsNullOrEmpty(homeDir))
             {
-                if (string.IsNullOrEmpty(string.Format("{0}\\{1}\\", Globals.ApplicationMapPath, homeDir).Replace("/", "\\")))
+                var fullHomeDir = string.Format("{0}\\{1}\\", Globals.ApplicationMapPath, homeDir).Replace("/", "\\");
+                if (Directory.Exists(fullHomeDir))
```

Comment:
Maybe this check should actually be done by reading HomeDirectory column from Portals table? @dnnsoftware/tag

Reply:
@tpluscode not sure it already exists at this point. Let me verify.